dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/README.md

```diff
@@ -665,6 +665,55 @@ var filtered = myCollection
 
 Maximum allowed clauses can be configured in `.editorconfig` by setting `dotnet_diagnostic.ACL1012.max_where_clauses`.
 
+## ACL1013 - Use record types
+
+<table>
+<tr>
+    <td>Category:</td>
+    <td>Usage</td>
+</tr>
+<tr>
+    <td>Audacia coding standard:</td>
+    <td>CS-02.9</td>
+</tr>
+</table>
+
+ACL1013 checks for classes that are suitable to be replaced with record types in projects with a language version of >= C#9. It will raise a warning if a type with a particular suffix is declared as a `class`.
+
+Code with diagnostic:
+
+```csharp
+public class PersonDto
+{
+    public string Name { get; set; }
+    public int Age { get; set; }
+}
+```
+
+Code without diagnostic:
+
+```csharp
+public record PersonDto
+{
+    public string Name { get; set; }
```

Comment:
Apologies if this is off topic and needs a separate issue raising.

1. Should the `set;` be `init;`?
2. Should we have a rule for properties in a record that aren't using positional they should have `init;` not `set;`?

Reply:
You mean CS-03.10? Yeah good point, that doesn't adhere to our standards. I'll update the example, but to answer your second question I think @richardb355 would prefer a work item raised [here](https://dev.azure.com/audacia/Audacia/_boards/board/t/Internal%20Development/Backlog%20items) to capture that analyzer.

That being said, I'm not sure how we'd enforce that with an analyzer? But happy to raise that when the PBI is raised :)

Reply:
Happy to create a work item if that's how we would like to proceed.

Might be worth us having an internal conversation as to why we would allow having properties with `set;` within an record. 

But... in the standards the example does have properties with `set;`, which maybe could be updated with the guidance of all records should have `get; init;` and if a property requires an update we should use the `with` expression? 

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/with-expression

Reply:
Yeah let's move this discussion to the work item if you're OK to create it, I'll respond there :)